cmake/compile_and_update_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script can modify the test results in the source
+# directory to the official tester results, if docker is installed
+# on the system. Start this script by running the following command
+# after replacing ASPECT_SOURCE_DIR with the directory of your ASPECT
+# folder:
+#
+# docker run -v ASPECT_SOURCE_DIR:/home/dealii/aspect --name=aspect-tester --rm -it dealii/dealii:v8.5.0-gcc-mpi-fulldepscandi-debugrelease bash /home/dealii/aspect/cmake/compile_and_update_tests.sh
```

Comment:
would it make sense to ask the user to execute this command in the source folder and then just specify ``$PWD`` here?

Reply:
I do not have a preference. Both is possible. In the current form it requires some user action, but it allows running it from everywhere. Do you have a preference?

Reply:
I prefer something copy&paste-able, but I don't feel strongly about it.